tests/staggered_invert_test.cpp

```diff
@@ -243,6 +243,9 @@ int main(int argc, char **argv)
   }
 
   // Create ghost gauge fields in case of multi GPU builds.
+  gauge_param.type = (dslash_type == QUDA_STAGGERED_DSLASH || dslash_type == QUDA_LAPLACE_DSLASH) ?
+    QUDA_SU3_LINKS :
+    QUDA_ASQTAD_FAT_LINKS;
```

Comment:
Just curious, (and sorry for the duplicate comment/question), Was this breaked at some point, or it's broken from day one?

Reply:
From Evan's comments to me last night, I believe this has been broken for years.